### DISCUSSION TITLE
Special Environmental Variables

### DISCUSSION THREAD

**Discussion by uncomfyhalomacro** [helix-editor/helix#7127] [Ideas]
This is a follow up for #7105. As mentioned on that issue, this is most inspired from kakoune's way of using [shell-expansions](https://manpages.debian.org/testing/kakoune/kak_expansions.1.en.html#SHELL_EXPANSIONS). 

For me it is a good feature. However, I am not sure what special environmental variables should be necessary to have if we go for this feature. My use case for this is to configure LSPs that have weird configurations e.g. Julia. 

The question is, do any of you even want this?

**Comment by guissalustiano** [comment]
Yes, I discovered this on Zed (https://github.com/zed-industries/zed/pull/8675) and it is pretty useful to run test in the current cursor in Elixir with
```
iex -S" mix test -b $ZED_FILE:$ZED_ROW
```
I would like to use this on helix too

**Reply by kirawi** [reply]
See https://github.com/helix-editor/helix/pull/6979
